client/my-sites/checkout/concierge-quickstart-session/index.jsx

```diff
@@ -361,38 +372,36 @@ export class ConciergeQuickstartSession extends React.Component {
 	};
 
 	handleClickAccept = () => {
-		const { siteSlug, trackUpsellButtonClick } = this.props;
-
-		trackUpsellButtonClick( 'accept' );
```

Comment:
It makes more sense to me to keep the click events tied directly to the users click action, rather than it being implied later on. Visiting or being redirected to the checkout URL where an item is automatically added to the cart shouldn't be confused with a click.

We should also probably track a different event when a user clicks the get started button, versus the yes... button, so we have more detailed data around what a user is actually clicking.

Reply:
I have implemented this in [98e7cf5](https://github.com/Automattic/wp-calypso/pull/33749/commits/98e7cf52d5be5495a3cbc58090bc419896580863). I've also added a tracking for the get started button.